contracts/base/Dispatcher.sol

```diff
@@ -210,8 +211,11 @@ abstract contract Dispatcher is Payments, V2SwapRouter, V3SwapRouter, V3ToV4Migr
                 }
                 // 0x10 <= command < 0x18
             } else {
-                // This contract MUST be approved to spend the token since its going to be doing the call on the position manager
-                if (command == Commands.V3_POSITION_MANAGER_PERMIT) {
+                if (command == Commands.V4_SWAP) {
+                    // pass the calldata provided to V4SwapRouter._executeActions (defined in BaseActionsRouter)
+                    _executeActions(inputs);
+                    // This contract MUST be approved to spend the token since its going to be doing the call on the position manager
```

Comment:
super nit: can we actually move this comment under the V3_POSITION_MANAGER_PERMIT

Reply:
Also I think it should be "permitted" since that is likely how we will do migration right?

Reply:
anyways unrelated to this PR... looks great!